Lombiq.Hosting.MultiTenancy.Tenants/Manifest.cs

```diff
@@ -0,0 +1,17 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Lombiq Hosting MultiTenancy Tenants",
+    Author = "The Orchard Core Team",
+    Website = "https://github.com/Lombiq/Hosting-Tenants",
+    Version = "0.0.1",
+    Description = "Adds useful features for multi-tenant applications.",
+    Category = "Hosting"
+)]
+
+[assembly: Feature(
+    Id = "FeaturesGuard",
```

Comment:
Feature Id should be the module namespace + feature name, so in this case Lombiq.Hosting.MultiTenancy.Tenants.FeaturesGuard, you could create a static class for this.